pangeo-deploy/templates/pangeo-rbac.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.rbac.enabled -}}
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: pangeo
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: pangeo
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+{{- end -}}
```

Comment:
Should this replace the `dask-kubernetes` service account in our helm chart? https://github.com/pangeo-data/helm-chart/blob/master/pangeo/templates/dask-kubernetes-rbac.yaml

Reply:
I think so, but I didn't want to remove it yet in case folks want to keep it around for a bit

Reply:
I think we can remove it upstream. People can keep using old tags if they like.